src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -93,4 +102,25 @@ export class NavbarComponent implements OnInit {
     getImageUrl() {
         return this.isAuthenticated() ? this.accountService.getImageUrl() : null;
     }
+
+    /**
+     * Checks if the current component has a guided tour by comparing the current router url to manually defined urls
+     * that provide tours.
+     */
+    checkGuidedTourAvailability() {
+        if (this.router.url === '/overview') {
+            this.isTourAvailable = true;
+        } else {
+            this.isTourAvailable = false;
+        }
+    }
+
+    /**
+     * Starts the guided tour of the current component
+     * */
+    startGuidedTour() {
+        if (this.router.url === '/overview') {
```

Comment:
Call ` checkGuidedTourAvailability()` instead of `this.router.url === '/overview'`